src/include/core.h

```diff
@@ -153,6 +153,9 @@ struct ncclRing {
       int* userRanks;
       int* devUserRanks;
 
+      // Next GPU's HDP register address
+      uint32_t* hdp_reg;
```

Comment:
Can we briefly explain here what the HDP register is used for?

Reply:
Done